recipes/cellsnp-lite/meta.yaml

```diff
@@ -20,7 +20,7 @@ requirements:
     - htslib
     - zlib
   run:
-    - htslib
+    - htslib >=1.9,<1.10.0a0
```

Comment:
We pin everything currently to a single htslib version (1.10 I believe). If this needs 1.9 then you'll need to add a conda_build_config.yaml specifying that, as it will then override the project-wide pinnings.

Reply:
ok, thanks!